depends/packages/packages.mk

```diff
@@ -0,0 +1,9 @@
+# zip is currently broken. Add it back to packages when it is fixed.
+packages:=gcc gmp mpc mpfr openssl psutil pyqt5 python3 rsync sip swig twisted zope.interface
```

Comment:
For now, please build toward Python 2 & Qt 4. We will be moving over eventually. I just don't know when. Until then, we need Py2/Qt4. Sorry!